docs/fundamentals/toc.yml

```diff
@@ -1791,6 +1791,8 @@ items:
       href: ../core/versions/index.md
     - name: .NET version selection
       href: ../core/versions/selection.md
+    - name: Compatibility
```

Comment:
I also just realized that the entire compatibility TOC is orphaned as of July 2021 (see https://github.com/dotnet/docs/pull/24630/files#r1054837339). I will figure out a place to slot it back into this TOC.